storage_service/common/utils.py

```diff
@@ -584,7 +584,13 @@ def removedirs(relative_path, base=None):
     """ Removes leaf directory of relative_path and all empty directories in
     relative_path, but nothing from base.
 
-    Cribbed from the implementation of os.removedirs. """
+    Cribbed from the implementation of os.removedirs.
+
+    :param relative_path: quad-dir structure,
+        e.g. aa9a/f3b1/32ae/4f3e/8841/0539/568c/43f2 (path-string)
+    :param base: root location of quad-dir structure,
+        e.g. /var/archivematica/storage_service (path-string, (optional))
```

Comment:
Your influence! I have some issues with support and consistency of docstrings in Python in general but I am increasingly seeing the advantage. And I think I am settling into a consistency with their formatting from our AIPscan work. And also for for a function like this which despite its seeming simplicity, just really needs to be as correct as it can be :ok_hand: seemed extra needed.